### DISCUSSION TITLE
Test the new WebView Adobe integration

### DISCUSSION THREAD

**Discussion by cbra-caa** [MicrosoftEdge/WebView2Feedback#4377] [Q&A] [answered]
For the upcoming Adobification of the pdf viewer I had hoped to try to preview the 'changes' via the msPDFSharedLibrary option mentioned in the [announcement](https://techcommunity.microsoft.com/t5/microsoft-edge-insider/microsoft-edge-and-adobe-partner-to-improve-the-pdf-experience/ba-p/3733481). 

I have tried enabling it through `new CoreWebView2EnvironmentOptions(additionalBrowserArguments: "--enable-features=msPDFSharedLibrary")` but is not seeing any effect. Does anyone else have any experience with it?

**Comment by victorhuangwq** [comment, marked as answer]
@monica-ch do you have any ideas of how to enable it?

**Reply by monica-ch** [reply]
@cbra-caa The feature flag should enable the new Adobe PDF viewer in WV2. However, you won't be able to see Adobe image on the bottom of the PDF like on edge browser for WV2 by design. You can check this WV2 app is using new PDF viewer from the Task manager processes of the host app.

**With current default viewer:**
![image](https://github.com/MicrosoftEdge/WebView2Feedback/assets/69160047/129c5e3d-dc7b-4f5f-bba2-e907f8bffa71)


**New PDF viewer:**
![image](https://github.com/MicrosoftEdge/WebView2Feedback/assets/69160047/82330a94-d198-4d2a-be99-fb4f447e6bab)
